UANodeManager.cs

```diff
@@ -34,6 +34,10 @@ public class UANodeManager : CustomNodeManager2
 
         private readonly string _wotNodeset = Path.Combine(Directory.GetCurrentDirectory(), "Nodesets", "Opc.Ua.WoT.NodeSet2.xml");
         private readonly bool _useWotNodeset = false;
+        
+        private readonly string _wotNodesetBinary = Path.Combine(Directory.GetCurrentDirectory(), "Nodesets", "Opc.Ua.WoT.PredefinedNodes.uanodes");

```

Comment:
Do we really need to support both binary and nodeset.xml import? If we just support nodeset.xml import, we can remove the dependency on model compiler, which is something I've been asked to do many times before.

Reply:
Currently yes, importing the NodeSet as xml will currently not map the input and output arguments, but importing the binary will.
If someone can tell me how the input- and outputArguments can be mapped from the xml without too much hassle or
once this issue is fixed in the stack (if ever?), I think we could and should drop the binary as you suggest.